File: 66-plus-one/plus-one.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        num=""
        for item in digits:
            num+=str(item)
        print(num)
        num=int(num)
        num+=1
        result=[]
        for item in str(num):
            result.append(int(item))
        return result

File: 66-plus-one/plus-one.js
/**
 * @param {number[]} digits
 * @return {number[]}
 */
var plusOne = function(digits) {
    let n = digits.length
    let i = n-1
    let val = 1
    while (true){
        let temp = digits[i]+val
        digits[i] = temp%10
        if (temp>=10){
            val = Math.floor(temp/10)
            i--
        }
        else{
            break
        }
    }
    if (i<0){
        digits.unshift(val)
    }
    return digits
};